modules/op-changing-source-branch-in-repository-crd.adoc

```diff
@@ -0,0 +1,25 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: REFERENCE
+[id="changing-source-branch-in-repository-crd_{context}"]
+= Changing the source branch for the pipeline definition in the `Repository` CRD
```

Comment:
Not an option because the Repository CRD is intentionally separate from the pipeline definition.

Reply:
I understand that, but the action being taken here by the user is "changing the source for the pipeline definition" - the CRD is _how_ they do that, not why, so I'm just suggesting that it doesn't need to be in the title. Up to you whether you leave it in though. The `Respository` does need to be changed to just Repository with no backticks in the title either way.

Reply:
Ok, I did go with removing the Repository CRD name from some module titles, removed backticks from the one module title where it remains, and expanded CRD the first time it is used in every module of this assembly (including modules otherwise unaffected by this PR)